packages/mobile/src/navigator/NavigationService.ts

```diff
@@ -88,6 +88,28 @@ export function navigate<RouteName extends keyof StackParamList>(
     })
 }
 
+export function navigateClearingStack<RouteName extends keyof StackParamList>(
+  ...args: undefined extends StackParamList[RouteName]
+    ? [RouteName] | [RouteName, StackParamList[RouteName]]
+    : [RouteName, StackParamList[RouteName]]
+) {
```

Comment:
nit: avoids the (complex) type repetition.
```suggestion
export const navigateClearingStack : SafeNavigate = (...args) => {
```